std/math.d

```diff
@@ -6577,60 +6566,50 @@ body
         if (c)
             m |= 0x4000_0000_0000_0000L; // shift carry into significand
         if (e)
-            *ul = m | 0x8000_0000_0000_0000L; // set implicit bit...
+            res.integral.significand = m | 0x8000_0000_0000_0000L; // set implicit bit...
         else
-            *ul = m; // ... unless exponent is 0 (subnormal or zero).
+            res.integral.significand = m; // ... unless exponent is 0 (subnormal or zero).
 
-        ue[4]= e | (xe[F.EXPPOS_SHORT]& 0x8000); // restore sign bit
+        res.shorts[4]= e | (repX.shorts[F.EXPPOS_SHORT]& 0x8000); // restore sign bit
```

Comment:
Nit: I can see that the previous code was this way, and you just exchanged ``ue`` and ``xe``, but...
```
res.shorts[4] = e | (repX.shorts[F.EXPPOS_SHORT] & 0x8000); // restore sign bit
             ^--- space added here, and also ---^
```